### PR TITLE
Sdk-Patfrom compatibility

### PR DESCRIPTION
**Pull request by RingerJK**

### CODE REVIEW

File: MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt [hunk 1]
```diff
@@ -135,21 +135,22 @@ internal class SensorsManager(application: Application) : SensorEventListener {
                             (value - gravity[index]) / SensorManager.GRAVITY_EARTH
                         }
                         .toFloatArray(),
-                    heading = heading.toFloat()
+                    heading = headingGeomagnetic.toFloat(),
```
Comment (yunikkk) [inline MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt]: Rename field itself to headingGeomagnetic?

File: MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt [hunk 2]
```diff
@@ -135,21 +135,22 @@ internal class SensorsManager(application: Application) : SensorEventListener {
...
+                    attitude = Attitude.Attitude_XMagneticNorthZVertical
                 )
             )
 
             listener.onHeadingData(
                 HeadingData(
-                    heading.toFloat(),
+                    headingTrue.toFloat(),
```
Comment (yunikkk) [inline MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt]: Rename field to headingTrue?

Comment (yunikkk) [reply MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt]: Ah its already!

File: MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt (hunk 1 above)
Comment (yunikkk) [reply MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt]: Plus - should we pass trueHeading here, not magnetic?

Comment (RingerJK) [reply MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt]: it's a heading and type is received in `attitude` field. 
I get that without geomagnetic sensor `SensorManager` doesn't work so we should always have `headingGeomagnetic `

Comment (yunikkk) [reply MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt]: Ok

**Pull request merged**